mcs/tools/gacutil/gacutil.exe.sources

```diff
@@ -1,2 +1,9 @@
 driver.cs
 ../security/StrongNameManager.cs
+../../../external/ikvm/reflect/*.cs
+../../../external/ikvm/reflect/Emit/*.cs
+../../../external/ikvm/reflect/Metadata/*.cs
```

Comment:
Just keep the files there but define NO_SYMBOL_WRITER and you can drop Mono.CompilerServices.SymbolWriter lib